source/community/release_notes/palm.rst

```diff
@@ -0,0 +1,45 @@
+.. _Open edX Palm Release:
+
+#####################
```

Comment:
I figure that you used the Olive release notes as a template, but could you actually update the heading symbols [to the ones in this guide](https://docs.openedx.org/en/latest/documentors/references/quick_reference_rst.html#headings)?

We're trying to stick to those heading symbols so that it'll be easier to compose and copy-paste rST documentation across the new docs. I'll open an issue to update the symbols in older release notes.

Reply:
Thanks for pointing that out, I wasn't aware of the style guide. I updated the headings.